README.md

```diff
@@ -358,13 +359,16 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [i18n.cr](https://github.com/vladfaust/i18n.cr) - Internationalization shard
  * [immutable](https://github.com/lucaong/immutable) - Implementation of thread-safe, persistent, immutable collections
  * [inflector.cr](https://github.com/phoffer/inflector.cr) - Singularize, pluralize, camelize, etc (port from ActiveSupport)
+ * [ishi](https://github.com/toddsundsted/ishi) - Graph plotting package with a small API powered by gnuplot
```

Comment:
What do you think about creating a new section for this entry? "Scientific", for instance?

Reply:
that sounds good to me!

Reply:
@veelenga how about moving it under "Science and Data analysis" since that already exists?

Reply:
Yeah, sounds good :+1:

Reply:
fixed in b5b24f2